go/signedexchange/signedexchange.go

```diff
@@ -51,6 +51,10 @@ func NewExchange(uri *url.URL, requestHeaders http.Header, status int, responseH
 	}, nil
 }
 
+func (e *Exchange) Payload() []byte {
```

Comment:
Hmm, this sounds unsafe since payload can be modified on the caller side. At least we should have comments.